src/Take.Elephant.Sql/SqlHelper.cs

```diff
@@ -93,6 +94,20 @@ public static string GetLiteralJoinConditionStatement(IDatabaseDriver databaseDr
 
         }
 
+        public static string GetIsNotNullStatement(IDatabaseDriver databaseDriver, string[] columns)
+        {
+            return GetSeparateColumnsStatement(databaseDriver,
+                  databaseDriver.GetSqlStatementTemplate(SqlStatement.And),
+                  columns,
+                  columns, (d, c, p) => databaseDriver.GetSqlStatementTemplate(SqlStatement.QueryIsNotNull).Format(
+                               new
+                               {
+                                   column = databaseDriver.ParseIdentifier(c)
+                               }));
+        }
+
+        public static string CombineAndEqualsWithIsNotNullStatement(IDatabaseDriver databaseDriver, string andEqualStatement, string isNotNullStatement) => $"{andEqualStatement} {databaseDriver.GetSqlStatementTemplate(SqlStatement.And)} {isNotNullStatement}";
```

Comment:
This could be applied to any type of statements, not only "and" and "is not null" , so you can rename to something like `GetCombinedAndStatement`, and parameters could be `firstStatement` and `secondStatement` (without any type "qualification").
Also, to prevent that eventual "OR" statements get evaluated on wrong order, put each statement in parenthesis.